packages/core/src/Provider/index.tsx

```diff
@@ -5,9 +5,12 @@ import HTML5Backend from 'react-dnd-html5-backend';
 import BlurGate from '../components/BlurGate';
 import Editor from '../Editor';
 import { ReduxProvider } from '../reduxConnect';
+import { DisplayModes } from '../actions/display';
 interface ProviderProps {
   editor: Editor;
   dndBackend?: BackendFactory;
+  blurGateDisabled?: boolean;
+  blurGateDefaultMode?: DisplayModes;
```

Comment:
we should probably also add this in `Editor` from 'react-page/editor` as its now recommended to use just this simple component. 

but there i would name it `displayModeDefault` or `defaultDisplayMode`

do you still need the `blurGateDisabled` ? is there another issue with it?

Reply:
Yp I'll add it there. Yeah, I need to disable it. Even the blurAllCells doesn't work for me. I don't want my bottom toolbar closing while I am clicking inside a different portal. ```defaultDisplayMode``` sounds good. I will also add ```blurGateDisabled``` there. K?

Reply:
@PeterKottas the blur should work through a portal. in fact, the bottom toolbar is a portal.

The BlurGate checks whether the click was inside the react-tree of the component.
But this has only been tested with the @react-page/editor, not with the old approach. I think i now why it is not working